pages/components/docs/core-concepts

```diff
@@ -0,0 +1,71 @@
+# Core Concepts
```

Comment:
```suggestion
# Core concepts
```

I think we're trying to be consistent about using sentence case for all of our titles. cc @ashygee